code/datums/religions.dm

```diff
@@ -738,6 +823,32 @@
 	male_adept = "Cultist"
 	female_adept = "Cultist"
 	keys = list("cult", "narsie", "nar'sie", "narnar")
+	convert_method = "performing a ritual with a paper. The subject will need to stand a crayon-drawn rune."
+
+/datum/religion/cult/convertCeremony(var/mob/living/preacher, var/mob/living/subject)
+	var/obj/effect/decal/cleanable/crayon/rune = locate(/obj/effect/decal/cleanable/crayon/, subject.loc)
+	if (!rune)
+		to_chat(preacher, "<span class='warning'>The subject needs to stand a crayon-drawn rune.</span>")
+		return FALSE
+	var/held_paper = preacher.find_held_item_by_type(/obj/item/weapon/paper)
+	if (!held_paper)
+		to_chat(preacher, "<span class='warning'>You need to hold a paper to begin to convert.</span>")
+		return FALSE
+
+	subject.visible_message("<span class='notice'>\The [preacher] attemps to convert \the [subject] to [name].</span>")
+
+	if(!convertCheck(subject))
+		subject.visible_message("<span class='warning'>\The [subject] refuses conversion.</span>")
+		return FALSE
+
+	if (prob(10))
+		preacher.say("DREAM SIGN: EVIL SEALING TALISMAN!")
+		subject.Knockdown(1)
+
+	sleep(0.2 SECONDS)
```

Comment:
why though?

Reply:
Simulate the fact that the cultist have to click on the rune.
I can put it behind the `if(prob(10))` if it's expensive

Reply:
not expensive at all, just seemed weird.